lms/templates/certificates/_certificate-supplemental.html

```diff
@@ -0,0 +1,36 @@
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
If it helps, this is the content placed below the certificate/achievement area. See this screengrab of our ID Verified cert UI.

![screen shot 2015-03-16 at 9 10 45 am](https://cloud.githubusercontent.com/assets/163763/6666618/68af64c4-cbbc-11e4-925f-d29bd551f060.png)